perception/mil_vision/mil_vision_tools/__init__.py

```diff
@@ -1,3 +1,5 @@
 from shape_finder import RectFinder, EllipseFinder, CircleFinder
-from cv_tools import Threshold, auto_canny
+from cv_tools import *
```

Comment:
Going for the blanket import? Bold strategy, Cotton. Lets see if it pays off for him.